prime-router/metadata/schemas/Experity/experity-covid-19.schema

```diff
@@ -47,11 +53,11 @@ elements:
     - name: patient_zip_code
       csvFields: [{name: Patient_zip_code}]
 
-    - name: patient_county
-      type: TABLE
-      table: zip-code-data
-      tableColumn: county
-      mapper: zipCodeToCounty(patient_zip_code)
+#    - name: patient_county
```

Comment:
I don't understand why you commented this out.  If you wanted remove then please delete from here.

Reply:
I commented this while I was testing moving the patient_county to the Child schema level.  I'll remove this now.